coronataiou/templates/base.html

```diff
@@ -14,53 +14,54 @@
     {% else %}
     <title>Galileo Temperature Manager</title>
     {% endif %}
-    <style>
-    body {
-    padding-top: 60px;
-    }
```

Comment:
does this render properly without the padding?

Reply:
yes it does render properly